app/export.go

```diff
@@ -148,7 +155,7 @@ func (app *UmeeApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs [
 	counter := int16(0)
 
 	for ; iter.Valid(); iter.Next() {
-		addr := sdk.ValAddress(iter.Key()[1:])
+		addr := sdk.ValAddress(stakingtypes.AddressFromValidatorsKey(iter.Key()))
```

Comment:
before it was `[1:]` right now AddressFromValidatorsKey uses `[2:]`. Is this expected?

Reply:
yes, before it was not fully corrected (it worked because prefixes are longer)